zh-CN/mapping_dev.zh_CN

```diff
@@ -1,44 +1,44 @@
 :*"Cancel"=取消
-:*"Add Branch"=创建分支
+:*"Add Branch"=添加分支
 :*"Discard"=丢弃
-:*"Git-Flow"=Git工作流
+:*"Git-Flow"=Git 流
 :*"Commit"=提交
 :*"Branch"=分支
 :*"Check Out"=检出
 :*"Merge"=合并
 :*"Pull"=拉取
 :*"Push"=推送
 :*"Rebase"=变基
-:*"Save Stash"=贮藏
+:*"Save Stash"=保存储藏
```

Comment:
Between "贮藏" and "储藏", I think "贮藏" is more appropriate, Not only because of Chinese interpretation, but also consistent with other git GUIs.

I will not point out one by one in other places, please confirm and update.

Reply:
Thanks, I've fixed it.